templates/policygen/org_policies/main.tf

```diff
@@ -18,6 +18,10 @@
 
 {{- $type_field := printf "%s_id" .ROOT_TYPE}}
 
+terraform {
```

Comment:
meant to put the comment here for terragrunt leaf

Reply:
https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/tree/master/templates/tfengine/components/terragrunt/leaf

Reply:
Done